.env.example

```diff
@@ -1,4 +1,7 @@
 JWT_SECRET='super_secret'
 JWT_EXPIRES_IN='24h'
-
+DATABASE_URL=database_url
+DIRECT_URL=example_url
+SUPABASE_URL=https://example.supabase.co
```

Comment:
We probably need to think about the way to securely share creds to DB with our FE team, or setup stage env for them to test endpoints.

Reply:
I don't think that we need this for this project. 
Even if we will have a stage we still need to share them

Reply:
https://www.dotenv.org/docs/quickstart#initial-setup. Maybe we can try this?

Reply:
Yeah, we can, but this will require our FE team to setup their own Supabase accounts and seed DB with data.
Thus, could complicate things out.

Reply:
Do you want me to do the stage or what? I don't want to manage 2 databases for this small project. 
I do not think it is imprortant for now.

Reply:
That's the idea to not manage several DBs, but have one that code be sharable between all folks in a team, but this require either setting up a stage (more secure, but more hustle) or sharing Supabase creds (not very secure, but easier).

Another option is to let all folks to create their own Supabase accounts, use them for each of individual dev envs and for BE team to additionally create a seeding mechanism to populate DB with testing data.

Reply:
As we agreed that you'll do the stage I resolve that